osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoom.cs

```diff
@@ -275,14 +275,6 @@ public void OnReleased(GlobalAction action)
 
         protected override bool ShouldBeConsideredForInput(Drawable child) => state == SelectionState.Selected;
 
-        protected override bool OnMouseDown(MouseDownEvent e)
```

Comment:
This used to be here specifically for popover container usage I think. Seems to work fine without now.